### ISSUE TITLE
Using `StarDist` without `pip install`

### ISSUE THREAD

**Issue by ajinkya-kulkarni** [stardist/stardist#250]
Hello team, I was trying to try out the conic model (`https://github.com/stardist/stardist/tree/conic-2022/examples/conic-2022`) on my custom dataset. I tried several times installing the conic branch using pip install. I was wondering if I can simply use the conic branch without a pip install. Like locally importing the necessary libraries after downloading the code. 

Thanks!

**Comment by ajinkya-kulkarni** [comment]
Yes, I read the troubleshooting section as well. This is how I tried to install the `conic` branch:

`conda create stardist-conic python=3.9` followed by `brew install gcc` and then `CC=gcc-10 CXX=g++-10 pip install git+https://github.com/stardist/stardist.git@conic-2022`.
But this gives an error and the installation stops. Am I doing anything wrong? I am using a macbook, but I get the same error on a Ubuntu machine.

**Comment by uschmidt83** [comment]
> `conda create stardist-conic python=3.9` followed by `brew install gcc` and then `CC=gcc-10 CXX=g++-10 pip install git+https://github.com/stardist/stardist.git@conic-2022`.

On my Mac, `brew install gcc` tries to install gcc version 13, i.e. `gcc-10` / `g++-10` have to be changed to `gcc-13` / `g++-13`. You can install a specific gcc version like this: `brew install gcc@13`. (The same applies to linux.)

This might work:

```
brew install gcc@13
conda create stardist-conic python=3.9
# ...
conda activate stardist-conic
CC=gcc-13 CXX=g++-13 pip install git+https://github.com/stardist/stardist.git@conic-2022
```

> But this gives an error and the installation stops.

It would have been helpful to see the error message.

> I am using a macbook

Despite what I wrote above, compiling on Mac might also work without installing gcc first. I.e. try this:

```
brew install libomp  # optional to speed up some computations
conda create stardist-conic python=3.9
# ...
conda activate stardist-conic
pip install git+https://github.com/stardist/stardist.git@conic-2022
```

**Comment by ajinkya-kulkarni** [comment]
Thanks, I will try your approach now. In the meantime, is there a way you can share the conic trained model, so it will be easier for me to test it on custom HE image?

**Comment by ajinkya-kulkarni** [comment]
Hello @uschmidt83, I was able to pip install the `conic` branch on a Ubuntu machine, but only the CPU version. The training for `1000` epochs as mentioned in the train notebook would roughly take one month to complete on my CPU workstation.
It would therefore be great if you could share/upload the trained model as well, as I can test it out on my colon HE images, instead of waiting for a month for the training to finish.

Best, Ajinkya

**Comment by maweigert** [comment]
> The training for `1000` epochs as mentioned in the train notebook would roughly take one month to complete on my CPU workstation.

There seems to be a problem with `tensorflow` not recognizing your GPU. You can check this with 

```python 
tf.test.is_gpu_available()
```

**Comment by kwiechen** [comment]
Is it possible to merge stardist-conic with main?
